be/src/runtime/memory/thread_mem_tracker_mgr.h

```diff
@@ -95,7 +95,9 @@ class ThreadMemTrackerMgr {
     void consume(int64_t size, int skip_large_memory_check = 0);
     void flush_untracked_mem();
 
-    bool try_reserve(int64_t size);
+    bool try_reserve(int64_t size, std::string& err_msg);
+    bool try_reserve(int64_t size, bool print_err_msg = false);
```

Comment:
Do not need 2 duplicate method. Just return status instead of bool.
And the caller could print the status stack if he want.

Reply:
done